### ISSUE TITLE
Failed making account

### ISSUE THREAD

**Issue by HamzMas** [Tustin/BlazinRewardsBot#2]
Whenever i run the bot i am met with the error: Failed making account: The reCaptcha has not been resolved properly.

**Comment by Jolkant** [comment]
Yup, generator won't work again unless someone makes a Captcha bypass.

**Comment by DerCremaxxs** [comment]
rip

**Comment by EthanC** [comment]
Old issue, could probably close, new version is out.
